File: Source/ArrayRelatedOperations/find_most_frequent.js
function findMostFrequent(arr) {
    var maxCount = 0, res = -1;

    for (var i = 0; i < arr.length; i++) {
        var count = 1;
        for (var j = i + 1; j < arr.length; j++)
            if (arr[i] == arr[j])
                count++;
        if (count > maxCount) {
            maxCount = count;
            res = arr[i];
        }
    }

    return res;
}

File: Source/ArrayRelatedOperations/find_most_frequent.c
int find_most_frequent(int arr[], int n) {
    int max_count = 0, res = -1;

    for (int i = 0; i < n; i++) {
        int count = 1;
        for (int j = i + 1; j < n; j++)
            if (arr[i] == arr[j])
                count++;
        if (count > max_count) {
            max_count = count;
            res = arr[i];
        }
    }

    return res;
}
